src/Tomcat/TomcatController.ts

```diff
@@ -230,34 +242,22 @@ export class TomcatController {
     }
 
     private async startTomcat(serverInfo: TomcatServer, appName: string): Promise<void> {
-        let statusBar: vscode.StatusBarItem;
-        let statusBarCommand: vscode.Disposable;
         const serverName: string = serverInfo.getName();
         let watcher: chokidar.FSWatcher;
-        const serverUri: string = await this.getServerUri(serverInfo, appName);
         const serverConfig: string = serverInfo.getServerConfigPath();
         const serverPort: string = await Utility.getPort(serverConfig, Constants.PortKind.Server);
         const httpPort: string = await Utility.getPort(serverConfig, Constants.PortKind.Http);
         const httpsPort: string = await Utility.getPort(serverConfig, Constants.PortKind.Https);
 
         try {
-            if (serverUri) {
-                statusBar = vscode.window.createStatusBarItem();
-                statusBar.command = `open.${serverName}`;
-                statusBar.text = `$(browser) Open ${appName}`;
-                statusBar.tooltip = localize('tomcatExt.open', 'Open: {0}', serverUri);
-                statusBarCommand = vscode.commands.registerCommand(statusBar.command, async () => {
-                    opn(serverUri);
-                });
-                statusBar.show();
-            }
-
             watcher = chokidar.watch(serverConfig);
             watcher.on('change', async () => {
                 if (serverPort !== await Utility.getPort(serverConfig, Constants.PortKind.Server)) {
                     vscode.window.showErrorMessage(localize('tomcatExt.serverPortChangeError', `Changing the server port of a running server will cause errors, please change it back to ${ serverPort} ！`));
```

Comment:
Looks like we have a wrong `！`here

Reply:
the prompt message was moved to DialogMessage in another PR, will update the branch after another PR merged.